src/hotspot/share/gc/g1/g1CollectedHeap.hpp

```diff
@@ -622,6 +622,8 @@ class G1CollectedHeap : public CollectedHeap {
   // for all regions.
   void verify_region_attr_remset_is_tracked() PRODUCT_RETURN;
 
+  void clear_region_in_prev_bitmap(HeapRegion* hr);
```

Comment:
A name like `clear_prev_bitmap_for_region` would be preferable to me, putting what we clear right next to the verb.

Reply:
I prefer this as well.